webapp/platform/components/src/generic_modal/generic_modal.tsx

```diff
@@ -128,7 +128,7 @@ export class GenericModal extends React.PureComponent<Props, State> {
                 <button
                     autoFocus={this.props.autoFocusConfirmButton}
                     type='submit'
-                    className={classNames('GenericModal__button', isConfirmOrDeleteClassName, this.props.confirmButtonClassName, {
```

Comment:
This is going to break some e2e tests (because we are searching for class name, which we shouldn't). If you feel brave, you can try to fix the e2e tests (hopefully is just about changing the selectors for `GenericModal__button` for selectors for `btn-primary`, but maybe not).

Another option, is to keep these classes, and remove the definitions from the scss files so these are "empty classes", and file a ticket to remove and fix the e2e tests, so they are done properly looking for the correct thing.

Reply:
Added them back again.